eval/hf_runner.py

```diff
@@ -111,6 +112,21 @@ def run_hf_eval(args):
     # eos_token_id = tokenizer.convert_tokens_to_ids(["```"])[0]
     pipe = pipeline("text-generation", model=model, tokenizer=tokenizer)
 
+    support_beam_search = True
+    try:
+        pipe("Hi", num_beams=2, do_sample=False)
```

Comment:
Ideally, we should add a `max_new_tokens=5` here. This will prevent the model from spending too long in generate outputs. I'll do it as a separate PR after this PR is merged.